<!doctype html><html lang=en-us><head><meta charset=utf-8><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><!--[if IEMobile]><meta http-equiv=cleartype content="on"><![endif]--><title>Proceedings of the BioNLP 2018 workshop - ACL Anthology</title><meta name=generator content="Hugo 0.98.0"><link href=/aclicon.ico rel="shortcut icon" type=image/x-icon><link rel=stylesheet href=/css/main.min.b1d14a9a8f6bb9c608ca4de9aad72a6e06945119f97951f2908522dc220e6277.css media=screen><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.7.2/css/all.css integrity=sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr crossorigin=anonymous><link rel=stylesheet href=/css/academicons.min.css></head><body><nav class="navbar navbar-expand-sm navbar-dark bg-dark bg-gradient-dark shadow-sm py-0 mb-3 mb-md-4 mb-xl-5"><div id=navbar-container class=container><a class=navbar-brand href=/><img src=/images/acl-logo.svg width=56 alt="ACL Logo">
<span class="d-none d-md-inline pl-md-2">ACL 2022 D&I Special Initiative</span></a>
<button class=navbar-toggler type=button data-toggle=collapse data-target=#navbarSupportedContent aria-controls=navbarSupportedContent aria-expanded=false aria-label="Toggle navigation">
<span class=navbar-toggler-icon></span></button><div class="collapse navbar-collapse" id=navbarSupportedContent><ul class="navbar-nav mr-auto"><li class=nav-item><a class=nav-link href=/>Papers<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/terms/>Terminology<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/>Videos<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/info/corrections/>Corrections<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/info/credits/>Credits<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/faq/>FAQ<span class=sr-only>(current)</span></a></li></ul><form class="form-inline my-2 my-lg-0" action=/search/? method=get><input id=acl-search-box class="form-control mr-sm-2" name=q type=search placeholder=Search... aria-label=Search>
<button class="btn btn-outline-secondary" type=submit><i class="fas fa-search"></i></button></form></div></div></nav><div id=main-container class=container><section id=main><h2 id=title><a href=https://aclanthology.org/W18-23.pdf>Proceedings of the <span class=acl-fixed-case>B</span>io<span class=acl-fixed-case>NLP</span> 2018 workshop</a></h2><p class=lead><a href=/people/d/dina-demner-fushman/>Dina Demner-Fushman</a>,
<a href=/people/k/k-bretonnel-cohen/>Kevin Bretonnel Cohen</a>,
<a href=/people/s/sophia-ananiadou/>Sophia Ananiadou</a>,
<a href=/people/j/junichi-tsujii/>Junichi Tsujii</a>
<span class=text-muted>(Editors)</span><hr><div class="row acl-paper-details"><div class="col col-lg-10 order-2"><dl><dt>Anthology ID:</dt><dd>W18-23</dd><dt>Month:</dt><dd>July</dd><dt>Year:</dt><dd>2018</dd><dt>Address:</dt><dd>Melbourne, Australia</dd><dt>Venues:</dt><dd><a href=/venues/acl/>ACL</a>
| <a href=/venues/bionlp/>BioNLP</a>
| <a href=/venues/ws/>WS</a></dd><dt>SIG:</dt><dd></dd><dt>Publisher:</dt><dd>Association for Computational Linguistics</dd><dt>URL:</dt><dd><a href=https://aclanthology.org/W18-23>https://aclanthology.org/W18-23</a></dd><dt>DOI:</dt><dd></dd><dt class=acl-button-row>Bib Export formats:</dt><dd class=acl-button-row></dd><dt>PDF:</dt><dd><a href=https://aclanthology.org/W18-23.pdf>https://aclanthology.org/W18-23.pdf</a></dd></dl></div><div class=acl-paper-link-block><a class="btn btn-primary" href=https://aclanthology.org/W18-23.pdf title="Open PDF of 'Proceedings of the BioNLP 2018 workshop'"><i class="far fa-file-pdf"></i><span class=pl-2>PDF&nbsp;<small>(full)</small></span></a>
<a class="btn btn-secondary" href="https://www.semanticscholar.org/search?q=Proceedings+of+the+BioNLP+2018+workshop" title="Search for 'Proceedings of the BioNLP 2018 workshop' on Semantic Scholar"><i class="ai ai-semantic-scholar"></i><span class="pl-sm-2 d-none d-sm-inline">Search</span></a></div></div><hr><div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W18-2300.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W18-2300/>Proceedings of the <span class=acl-fixed-case>B</span>io<span class=acl-fixed-case>NLP</span> 2018 workshop</a></strong><br><a href=/people/d/dina-demner-fushman/>Dina Demner-Fushman</a>
|
<a href=/people/k/k-bretonnel-cohen/>Kevin Bretonnel Cohen</a>
|
<a href=/people/s/sophia-ananiadou/>Sophia Ananiadou</a>
|
<a href=/people/j/junichi-tsujii/>Junichi Tsujii</a></span></p><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W18-2301.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W18-2301 data-toggle=collapse aria-expanded=false aria-controls=abstract-W18-2301 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"><a class="badge badge-secondary align-middle mr-1 pwc-reduced-padding" href="https://paperswithcode.com/paper/?acl=W18-2301" data-toggle=tooltip data-placement=top title=Code><svg xmlns="http://www.w3.org/2000/svg" class="pwc-icon-small" viewBox="0 0 512 512"><path stroke="#4d8093" fill="#4d8093" d="M88 128h48v256H88zm144 0h48v256h-48zm-72 16h48v224h-48zm144 0h48v224h-48zm72-16h48v256h-48z"/><path stroke="#4d8093" fill="#4d8093" d="M104 104V56H16v4e2h88v-48H64V104zM408 56v48h40v304h-40v48h88V56z"/></svg></a></span><span class=d-block><strong><a class=align-middle href=/W18-2301/>Embedding Transfer for Low-Resource Medical Named Entity Recognition : A Case Study on Patient Mobility</a></strong><br><a href=/people/d/denis-newman-griffis/>Denis Newman-Griffis</a>
|
<a href=/people/a/ayah-zirikly/>Ayah Zirikly</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W18-2301><div class="card-body p-3 small">Functioning is gaining recognition as an important indicator of global health, but remains under-studied in medical natural language processing research. We present the first analysis of automatically extracting descriptions of patient mobility, using a recently-developed <a href=https://en.wikipedia.org/wiki/Data_set>dataset</a> of free text electronic health records. We frame the task as a named entity recognition (NER) problem, and investigate the applicability of NER techniques to mobility extraction. As <a href=https://en.wikipedia.org/wiki/Text_corpus>text corpora</a> focused on patient functioning are scarce, we explore domain adaptation of <a href=https://en.wikipedia.org/wiki/Word_embedding>word embeddings</a> for use in a recurrent neural network NER system. We find that embeddings trained on a small in-domain corpus perform nearly as well as those learned from large out-of-domain corpora, and that domain adaptation techniques yield additional improvements in both precision and recall. Our analysis identifies several significant challenges in extracting descriptions of patient mobility, including the length and complexity of annotated entities and high linguistic variability in mobility descriptions.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W18-2304.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W18-2304 data-toggle=collapse aria-expanded=false aria-controls=abstract-W18-2304 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W18-2304/>Keyphrases Extraction from User-Generated Contents in Healthcare Domain Using Long Short-Term Memory Networks</a></strong><br><a href=/people/i/ilham-fathy-saputra/>Ilham Fathy Saputra</a>
|
<a href=/people/r/rahmad-mahendra/>Rahmad Mahendra</a>
|
<a href=/people/a/alfan-farizki-wicaksono/>Alfan Farizki Wicaksono</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W18-2304><div class="card-body p-3 small">We propose keyphrases extraction technique to extract important terms from the healthcare user-generated contents. We employ deep learning architecture, i.e. Long Short-Term Memory, and leverage <a href=https://en.wikipedia.org/wiki/Word_embedding>word embeddings</a>, medical concepts from a <a href=https://en.wikipedia.org/wiki/Knowledge_base>knowledge base</a>, and linguistic components as our <a href=https://en.wikipedia.org/wiki/Feature_(machine_learning)>features</a>. The proposed <a href=https://en.wikipedia.org/wiki/Mathematical_model>model</a> achieves 61.37 % F-1 score. Experimental results indicate that our proposed approach outperforms the <a href=https://en.wikipedia.org/wiki/Baseline_(medicine)>baseline methods</a>, i.e. RAKE and CRF, on the task of extracting keyphrases from Indonesian health forum posts.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W18-2305.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W18-2305 data-toggle=collapse aria-expanded=false aria-controls=abstract-W18-2305 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"><a class="badge badge-secondary align-middle mr-1 pwc-reduced-padding" href="https://paperswithcode.com/paper/?acl=W18-2305" data-toggle=tooltip data-placement=top title=Code><svg xmlns="http://www.w3.org/2000/svg" class="pwc-icon-small" viewBox="0 0 512 512"><path stroke="#4d8093" fill="#4d8093" d="M88 128h48v256H88zm144 0h48v256h-48zm-72 16h48v224h-48zm144 0h48v224h-48zm72-16h48v256h-48z"/><path stroke="#4d8093" fill="#4d8093" d="M104 104V56H16v4e2h88v-48H64V104zM408 56v48h40v304h-40v48h88V56z"/></svg></a></span><span class=d-block><strong><a class=align-middle href=/W18-2305/>Identifying Key Sentences for Precision Oncology Using <a href=https://en.wikipedia.org/wiki/Semi-supervised_learning>Semi-Supervised Learning</a></a></strong><br><a href=/people/j/jurica-seva/>Jurica Ševa</a>
|
<a href=/people/m/martin-wackerbauer/>Martin Wackerbauer</a>
|
<a href=/people/u/ulf-leser/>Ulf Leser</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W18-2305><div class="card-body p-3 small">We present a machine learning pipeline that identifies key sentences in abstracts of oncological articles to aid <a href=https://en.wikipedia.org/wiki/Evidence-based_medicine>evidence-based medicine</a>. This problem is characterized by the lack of gold standard datasets, data imbalance and thematic differences between available silver standard corpora. Additionally, available training and target data differs with regard to their domain (professional summaries vs. sentences in abstracts). This makes <a href=https://en.wikipedia.org/wiki/Supervised_learning>supervised machine learning</a> inapplicable. We propose the use of two semi-supervised machine learning approaches : To mitigate difficulties arising from heterogeneous data sources, overcome data imbalance and create reliable training data we propose using <a href=https://en.wikipedia.org/wiki/Transductive_learning>transductive learning</a> from positive and unlabelled data (PU Learning). For obtaining a realistic <a href=https://en.wikipedia.org/wiki/Statistical_classification>classification model</a>, we propose the use of <a href=https://en.wikipedia.org/wiki/Abstract_(summary)>abstracts</a> summarised in relevant sentences as unlabelled examples through Self-Training. The best <a href=https://en.wikipedia.org/wiki/Mathematical_model>model</a> achieves 84 % <a href=https://en.wikipedia.org/wiki/Accuracy_and_precision>accuracy</a> and 0.84 <a href=https://en.wikipedia.org/wiki/F-number>F1 score</a> on our <a href=https://en.wikipedia.org/wiki/Data_set>dataset</a></div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W18-2310.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W18-2310 data-toggle=collapse aria-expanded=false aria-controls=abstract-W18-2310 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W18-2310/>A Neural Autoencoder Approach for Document Ranking and Query Refinement in Pharmacogenomic Information Retrieval</a></strong><br><a href=/people/j/jonas-pfeiffer/>Jonas Pfeiffer</a>
|
<a href=/people/s/samuel-broscheit/>Samuel Broscheit</a>
|
<a href=/people/r/rainer-gemulla/>Rainer Gemulla</a>
|
<a href=/people/m/mathias-goschl/>Mathias Göschl</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W18-2310><div class="card-body p-3 small">In this study, we investigate learning-to-rank and query refinement approaches for <a href=https://en.wikipedia.org/wiki/Information_retrieval>information retrieval</a> in the <a href=https://en.wikipedia.org/wiki/Pharmacogenomics>pharmacogenomic domain</a>. The goal is to improve the information retrieval process of biomedical curators, who manually build <a href=https://en.wikipedia.org/wiki/Knowledge_base>knowledge bases</a> for <a href=https://en.wikipedia.org/wiki/Personalized_medicine>personalized medicine</a>. We study how to exploit the relationships between <a href=https://en.wikipedia.org/wiki/Gene>genes</a>, <a href=https://en.wikipedia.org/wiki/Single-nucleotide_polymorphism>variants</a>, <a href=https://en.wikipedia.org/wiki/Drug>drugs</a>, <a href=https://en.wikipedia.org/wiki/Disease>diseases</a> and outcomes as features for document ranking and query refinement. For a <a href=https://en.wikipedia.org/wiki/Supervised_learning>supervised approach</a>, we are faced with a small amount of <a href=https://en.wikipedia.org/wiki/Annotation>annotated data</a> and a large amount of unannotated data. Therefore, we explore ways to use a neural document auto-encoder in a <a href=https://en.wikipedia.org/wiki/Semi-supervised_learning>semi-supervised approach</a>. We show that a combination of established <a href=https://en.wikipedia.org/wiki/Algorithm>algorithms</a>, <a href=https://en.wikipedia.org/wiki/Feature_engineering>feature-engineering</a> and a neural auto-encoder model yield promising results in this setting.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W18-2311.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W18-2311 data-toggle=collapse aria-expanded=false aria-controls=abstract-W18-2311 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W18-2311/>Biomedical Event Extraction Using <a href=https://en.wikipedia.org/wiki/Convolutional_neural_network>Convolutional Neural Networks</a> and Dependency Parsing</a></strong><br><a href=/people/j/jari-bjorne/>Jari Björne</a>
|
<a href=/people/t/tapio-salakoski/>Tapio Salakoski</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W18-2311><div class="card-body p-3 small">Event and relation extraction are central tasks in <a href=https://en.wikipedia.org/wiki/Biomedical_text_mining>biomedical text mining</a>. Where <a href=https://en.wikipedia.org/wiki/Relation_extraction>relation extraction</a> concerns the detection of semantic connections between pairs of entities, <a href=https://en.wikipedia.org/wiki/Event_extraction>event extraction</a> expands this concept with the addition of trigger words, multiple arguments and nested events, in order to more accurately model the diversity of natural language. In this work we develop a <a href=https://en.wikipedia.org/wiki/Convolutional_neural_network>convolutional neural network</a> that can be used for both event and relation extraction. We use a linear representation of the input text, where information is encoded with various vector space embeddings. Most notably, we encode the <a href=https://en.wikipedia.org/wiki/Parse_graph>parse graph</a> into this <a href=https://en.wikipedia.org/wiki/Vector_space>linear space</a> using dependency path embeddings. We integrate our <a href=https://en.wikipedia.org/wiki/Neural_network>neural network</a> into the open source Turku Event Extraction System (TEES) framework. Using this <a href=https://en.wikipedia.org/wiki/System>system</a>, our <a href=https://en.wikipedia.org/wiki/Machine_learning>machine learning model</a> can be easily applied to a large set of corpora from e.g. the <a href=https://en.wikipedia.org/wiki/BioNLP>BioNLP</a>, DDI Extraction and BioCreative shared tasks. We evaluate our system on 12 different event, relation and NER corpora, showing good generalizability to many tasks and achieving improved performance on several corpora.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W18-2318.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W18-2318 data-toggle=collapse aria-expanded=false aria-controls=abstract-W18-2318 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W18-2318/>SingleCite : Towards an improved Single Citation Search in PubMed<span class=acl-fixed-case>S</span>ingle<span class=acl-fixed-case>C</span>ite: Towards an improved Single Citation Search in <span class=acl-fixed-case>P</span>ub<span class=acl-fixed-case>M</span>ed</a></strong><br><a href=/people/l/lana-yeganova/>Lana Yeganova</a>
|
<a href=/people/d/donald-c-comeau/>Donald C Comeau</a>
|
<a href=/people/w/won-kim/>Won Kim</a>
|
<a href=/people/w/w-john-wilbur/>W John Wilbur</a>
|
<a href=/people/z/zhiyong-lu/>Zhiyong Lu</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W18-2318><div class="card-body p-3 small">A <a href=https://en.wikipedia.org/wiki/Search_engine_technology>search</a> that is targeted at finding a specific document in databases is called a Single Citation search. Single citation searches are particularly important for scholarly databases, such as <a href=https://en.wikipedia.org/wiki/PubMed>PubMed</a>, because users are frequently searching for a specific publication. In this work we describe SingleCite, a single citation matching system designed to facilitate user&#8217;s search for a specific document. We report on the progress that has been achieved towards building that <a href=https://en.wikipedia.org/wiki/Function_(engineering)>functionality</a>.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W18-2319.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W18-2319 data-toggle=collapse aria-expanded=false aria-controls=abstract-W18-2319 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W18-2319/>A Framework for Developing and Evaluating Word Embeddings of Drug-named Entity</a></strong><br><a href=/people/m/mengnan-zhao/>Mengnan Zhao</a>
|
<a href=/people/a/aaron-j-masino/>Aaron J. Masino</a>
|
<a href=/people/c/christopher-c-yang/>Christopher C. Yang</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W18-2319><div class="card-body p-3 small">We investigate the quality of task specific word embeddings created with relatively small, targeted corpora. We present a comprehensive evaluation framework including both intrinsic and extrinsic evaluation that can be expanded to named entities beyond drug name. Intrinsic evaluation results tell that drug name embeddings created with a domain specific document corpus outperformed the previously published versions that derived from a very large general text corpus. Extrinsic evaluation uses <a href=https://en.wikipedia.org/wiki/Word_embedding>word embedding</a> for the task of drug name recognition with Bi-LSTM model and the results demonstrate the advantage of using domain-specific word embeddings as the only input <a href=https://en.wikipedia.org/wiki/Feature_(machine_learning)>feature</a> for drug name recognition with F1-score achieving 0.91. This work suggests that it may be advantageous to derive domain specific embeddings for certain tasks even when the domain specific corpus is of limited size.</div></div></div></section></div><footer class="bg-gradient-dark py-2 py-xl-3 mt-3 mt-md-4 mt-xl-5" style=color:#fff><div class=container><p class="small px-1"><span class="float-right mt-2 ml-2"><a rel=license href=http://creativecommons.org/licenses/by/4.0/><img alt="Creative Commons License" style=border-width:0 src=https://i.creativecommons.org/l/by/4.0/88x31.png></a></span>
ACL materials are Copyright ©&nbsp;1963&ndash;2022 ACL; other materials are copyrighted by their respective copyright holders. Materials prior to 2016 here are licensed under the <a href=https://creativecommons.org/licenses/by-nc-sa/3.0/>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 International License</a>. Permission is granted to make copies for the purposes of teaching and research. Materials published in or after 2016 are licensed on a <a href=https://creativecommons.org/licenses/by/4.0/>Creative Commons Attribution 4.0 International License</a>.</p><p class="small px-1">The ACL Anthology is managed and built by the <a href=/info/credits/>ACL Anthology team</a> of volunteers.</p><p class="small px-1"><i>Site last built on 23 May 2022 at 01:26 UTC with <a href=https://github.com/acl-org/acl-anthology/tree/6b5537cdc480294721047e4e67eca5cb2bec4ab4>commit 6b5537cd</a>.</i></p></div></footer><script src=https://code.jquery.com/jquery-3.3.1.slim.min.js integrity=sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js integrity=sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut crossorigin=anonymous></script>
<script src=https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js integrity=sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k crossorigin=anonymous></script>
<script>$(function(){$('[data-toggle="tooltip"]').tooltip(),$("#toggle-all-abstracts")&&($("#toggle-all-abstracts").click(function(){var e=$("#toggle-all-abstracts");e.attr("disabled",!0),e.attr("data-toggle-state")=="hide"?($(".abstract-collapse").collapse("show"),e.attr("data-toggle-state","show")):($(".abstract-collapse").collapse("hide"),e.attr("data-toggle-state","hide")),e.attr("disabled",!1)}),$("#toggle-all-abstracts").attr("disabled",!1))})</script></body></html>