<!doctype html><html lang=en-us><head><meta charset=utf-8><meta charset=utf-8><meta name=viewport content="width=device-width,initial-scale=1,shrink-to-fit=no"><!--[if IEMobile]><meta http-equiv=cleartype content="on"><![endif]--><title>BioNLP 2017 - ACL Anthology</title><meta name=generator content="Hugo 0.98.0"><link href=/aclicon.ico rel="shortcut icon" type=image/x-icon><link rel=stylesheet href=/css/main.min.b1d14a9a8f6bb9c608ca4de9aad72a6e06945119f97951f2908522dc220e6277.css media=screen><link rel=stylesheet href=https://use.fontawesome.com/releases/v5.7.2/css/all.css integrity=sha384-fnmOCqbTlWIlj8LyTjo7mOUStjsKC4pOpQbqyi7RrhN7udi9RwhKkMHpvLbHG9Sr crossorigin=anonymous><link rel=stylesheet href=/css/academicons.min.css></head><body><nav class="navbar navbar-expand-sm navbar-dark bg-dark bg-gradient-dark shadow-sm py-0 mb-3 mb-md-4 mb-xl-5"><div id=navbar-container class=container><a class=navbar-brand href=/><img src=/images/acl-logo.svg width=56 alt="ACL Logo">
<span class="d-none d-md-inline pl-md-2">ACL 2022 D&I Special Initiative</span></a>
<button class=navbar-toggler type=button data-toggle=collapse data-target=#navbarSupportedContent aria-controls=navbarSupportedContent aria-expanded=false aria-label="Toggle navigation">
<span class=navbar-toggler-icon></span></button><div class="collapse navbar-collapse" id=navbarSupportedContent><ul class="navbar-nav mr-auto"><li class=nav-item><a class=nav-link href=/>Papers<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/terms/>Terminology<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/>Videos<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/info/corrections/>Corrections<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/info/credits/>Credits<span class=sr-only>(current)</span></a></li><li class=nav-item><a class=nav-link href=/faq/>FAQ<span class=sr-only>(current)</span></a></li></ul><form class="form-inline my-2 my-lg-0" action=/search/? method=get><input id=acl-search-box class="form-control mr-sm-2" name=q type=search placeholder=Search... aria-label=Search>
<button class="btn btn-outline-secondary" type=submit><i class="fas fa-search"></i></button></form></div></div></nav><div id=main-container class=container><section id=main><h2 id=title><a href=https://aclanthology.org/W17-23.pdf><span class=acl-fixed-case>B</span>io<span class=acl-fixed-case>NLP</span> 2017</a></h2><p class=lead><a href=/people/k/k-bretonnel-cohen/>Kevin Bretonnel Cohen</a>,
<a href=/people/d/dina-demner-fushman/>Dina Demner-Fushman</a>,
<a href=/people/s/sophia-ananiadou/>Sophia Ananiadou</a>,
<a href=/people/j/junichi-tsujii/>Junichi Tsujii</a>
<span class=text-muted>(Editors)</span><hr><div class="row acl-paper-details"><div class="col col-lg-10 order-2"><dl><dt>Anthology ID:</dt><dd>W17-23</dd><dt>Month:</dt><dd>August</dd><dt>Year:</dt><dd>2017</dd><dt>Address:</dt><dd>Vancouver, Canada,</dd><dt>Venues:</dt><dd><a href=/venues/bionlp/>BioNLP</a>
| <a href=/venues/ws/>WS</a></dd><dt>SIG:</dt><dd><a href=/sigs/sigbiomed/>SIGBIOMED</a></dd><dt>Publisher:</dt><dd>Association for Computational Linguistics</dd><dt>URL:</dt><dd><a href=https://aclanthology.org/W17-23>https://aclanthology.org/W17-23</a></dd><dt>DOI:</dt><dd><a href=http://dx.doi.org/10.18653/v1/W17-23 title="To the current version of the paper by DOI">10.18653/v1/W17-23</a></dd><dt class=acl-button-row>Bib Export formats:</dt><dd class=acl-button-row></dd><dt>PDF:</dt><dd><a href=https://aclanthology.org/W17-23.pdf>https://aclanthology.org/W17-23.pdf</a></dd></dl></div><div class=acl-paper-link-block><a class="btn btn-primary" href=https://aclanthology.org/W17-23.pdf title="Open PDF of 'BioNLP 2017'"><i class="far fa-file-pdf"></i><span class=pl-2>PDF&nbsp;<small>(full)</small></span></a>
<a class="btn btn-secondary" href="https://www.semanticscholar.org/search?q=BioNLP+2017" title="Search for 'BioNLP 2017' on Semantic Scholar"><i class="ai ai-semantic-scholar"></i><span class="pl-sm-2 d-none d-sm-inline">Search</span></a></div></div><hr><div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2300.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2300/><span class=acl-fixed-case>B</span>io<span class=acl-fixed-case>NLP</span> 2017</a></strong><br><a href=/people/k/k-bretonnel-cohen/>Kevin Bretonnel Cohen</a>
|
<a href=/people/d/dina-demner-fushman/>Dina Demner-Fushman</a>
|
<a href=/people/s/sophia-ananiadou/>Sophia Ananiadou</a>
|
<a href=/people/j/junichi-tsujii/>Junichi Tsujii</a></span></p><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2301.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2301 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2301 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2301/>Target word prediction and paraphasia classification in spoken discourse</a></strong><br><a href=/people/j/joel-adams/>Joel Adams</a>
|
<a href=/people/s/steven-bedrick/>Steven Bedrick</a>
|
<a href=/people/g/gerasimos-fergadiotis/>Gerasimos Fergadiotis</a>
|
<a href=/people/k/kyle-gorman/>Kyle Gorman</a>
|
<a href=/people/j/jan-van-santen/>Jan van Santen</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2301><div class="card-body p-3 small">We present a system for automatically detecting and classifying phonologically anomalous productions in the <a href=https://en.wikipedia.org/wiki/Speech>speech</a> of individuals with <a href=https://en.wikipedia.org/wiki/Aphasia>aphasia</a>. Working from transcribed discourse samples, our system identifies <a href=https://en.wikipedia.org/wiki/Neologism>neologisms</a>, and uses a combination of string alignment and <a href=https://en.wikipedia.org/wiki/Language_model>language models</a> to produce a lattice of plausible words that the speaker may have intended to produce. We then score this <a href=https://en.wikipedia.org/wiki/Lattice_(order)>lattice</a> according to various features, and attempt to determine whether the anomalous production represented a phonemic error or a genuine <a href=https://en.wikipedia.org/wiki/Neologism>neologism</a>. This approach has the potential to be expanded to consider other types of paraphasic errors, and could be applied to a wide variety of screening and therapeutic applications.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2302.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2302 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2302 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2302/>Extracting Drug-Drug Interactions with Attention CNNs<span class=acl-fixed-case>CNN</span>s</a></strong><br><a href=/people/m/masaki-asada/>Masaki Asada</a>
|
<a href=/people/m/makoto-miwa/>Makoto Miwa</a>
|
<a href=/people/y/yutaka-sasaki/>Yutaka Sasaki</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2302><div class="card-body p-3 small">We propose a novel attention mechanism for a Convolutional Neural Network (CNN)-based Drug-Drug Interaction (DDI) extraction model. CNNs have been shown to have a great potential on DDI extraction tasks ; however, <a href=https://en.wikipedia.org/wiki/Attentional_control>attention mechanisms</a>, which emphasize important words in the sentence of a target-entity pair, have not been investigated with the CNNs despite the fact that <a href=https://en.wikipedia.org/wiki/Attentional_control>attention mechanisms</a> are shown to be effective for a general domain relation classification task. We evaluated our <a href=https://en.wikipedia.org/wiki/Mathematical_model>model</a> on the Task 9.2 of the DDIExtraction-2013 shared task. As a result, our attention mechanism improved the performance of our base CNN-based DDI model, and the <a href=https://en.wikipedia.org/wiki/Mathematical_model>model</a> achieved an F-score of 69.12 %, which is competitive with the state-of-the-art models.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2303.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2303 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2303 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"><a class="badge badge-secondary align-middle mr-1 pwc-reduced-padding" href="https://paperswithcode.com/paper/?acl=W17-2303" data-toggle=tooltip data-placement=top title=Code><svg xmlns="http://www.w3.org/2000/svg" class="pwc-icon-small" viewBox="0 0 512 512"><path stroke="#4d8093" fill="#4d8093" d="M88 128h48v256H88zm144 0h48v256h-48zm-72 16h48v224h-48zm144 0h48v224h-48zm72-16h48v256h-48z"/><path stroke="#4d8093" fill="#4d8093" d="M104 104V56H16v4e2h88v-48H64V104zM408 56v48h40v304h-40v48h88V56z"/></svg></a></span><span class=d-block><strong><a class=align-middle href=/W17-2303/>Insights into Analogy Completion from the Biomedical Domain</a></strong><br><a href=/people/d/denis-newman-griffis/>Denis Newman-Griffis</a>
|
<a href=/people/a/albert-m-lai/>Albert Lai</a>
|
<a href=/people/e/eric-fosler-lussier/>Eric Fosler-Lussier</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2303><div class="card-body p-3 small">Analogy completion has been a popular task in recent years for evaluating the semantic properties of word embeddings, but the standard methodology makes a number of assumptions about <a href=https://en.wikipedia.org/wiki/Analogy>analogies</a> that do not always hold, either in recent benchmark datasets or when expanding into other domains. Through an analysis of analogies in the biomedical domain, we identify three assumptions : that of a Single Answer for any given <a href=https://en.wikipedia.org/wiki/Analogy>analogy</a>, that the pairs involved describe the Same Relationship, and that each pair is Informative with respect to the other. We propose modifying the standard <a href=https://en.wikipedia.org/wiki/Methodology>methodology</a> to relax these assumptions by allowing for multiple correct answers, reporting MAP and MRR in addition to <a href=https://en.wikipedia.org/wiki/Accuracy_and_precision>accuracy</a>, and using multiple example pairs. We further present BMASS, a novel dataset for evaluating linguistic regularities in biomedical embeddings, and demonstrate that the relationships described in the dataset pose significant semantic challenges to current word embedding methods.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2304.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2304 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2304 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2304/>Deep learning for extracting protein-protein interactions from biomedical literature</a></strong><br><a href=/people/y/yifan-peng/>Yifan Peng</a>
|
<a href=/people/z/zhiyong-lu/>Zhiyong Lu</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2304><div class="card-body p-3 small">State-of-the-art methods for protein-protein interaction (PPI) extraction are primarily feature-based or kernel-based by leveraging lexical and syntactic information. But how to incorporate such knowledge in the recent <a href=https://en.wikipedia.org/wiki/Deep_learning>deep learning methods</a> remains an open question. In this paper, we propose a multichannel dependency-based convolutional neural network model (McDepCNN). It applies one channel to the <a href=https://en.wikipedia.org/wiki/Glossary_of_graph_theory_terms>embedding vector</a> of each word in the sentence, and another channel to the <a href=https://en.wikipedia.org/wiki/Glossary_of_graph_theory_terms>embedding vector</a> of the head of the corresponding word. Therefore, the <a href=https://en.wikipedia.org/wiki/Mathematical_model>model</a> can use richer information obtained from different channels. Experiments on two public benchmarking datasets, AIMed and BioInfer, demonstrate that McDepCNN provides up to 6 % F1-score improvement over rich feature-based methods and single-kernel methods. In addition, McDepCNN achieves 24.4 % relative improvement in <a href=https://en.wikipedia.org/wiki/F-number>F1-score</a> over the state-of-the-art methods on cross-corpus evaluation and 12 % improvement in <a href=https://en.wikipedia.org/wiki/F-number>F1-score</a> over kernel-based methods on difficult instances. These results suggest that McDepCNN generalizes more easily over different corpora, and is capable of capturing long distance features in the sentences.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2305.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2305 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2305 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2305/>Stacking With Auxiliary Features for Entity Linking in the Medical Domain</a></strong><br><a href=/people/n/nazneen-fatema-rajani/>Nazneen Fatema Rajani</a>
|
<a href=/people/m/mihaela-bornea/>Mihaela Bornea</a>
|
<a href=/people/k/ken-barker/>Ken Barker</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2305><div class="card-body p-3 small">Linking spans of natural language text to concepts in a structured source is an important task for many problems. It allows intelligent systems to leverage rich knowledge available in those sources (such as concept properties and relations) to enhance the <a href=https://en.wikipedia.org/wiki/Semantics>semantics</a> of the mentions of these concepts in text. In the medical domain, it is common to link text spans to medical concepts in large, curated knowledge repositories such as the <a href=https://en.wikipedia.org/wiki/Unified_Medical_Language_System>Unified Medical Language System</a>. Different approaches have different strengths : some are precision-oriented, some recall-oriented ; some better at considering context but more prone to <a href=https://en.wikipedia.org/wiki/Hallucination>hallucination</a>. The variety of techniques suggests that <a href=https://en.wikipedia.org/wiki/Assembly_language>ensembling</a> could outperform component technologies at this <a href=https://en.wikipedia.org/wiki/Task_(project_management)>task</a>. In this paper, we describe our process for building a Stacking ensemble using additional, auxiliary features for Entity Linking in the medical domain. We report experiments that show that naive ensembling does not always outperform component Entity Linking systems, that stacking usually outperforms naive ensembling, and that auxiliary features added to the stacker further improve its performance on three distinct datasets. Our best <a href=https://en.wikipedia.org/wiki/Mathematical_model>model</a> produces state-of-the-art results on several <a href=https://en.wikipedia.org/wiki/Data_set>medical datasets</a>.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2307.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2307 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2307 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2307/>Tackling Biomedical Text Summarization : OAQA at BioASQ 5B<span class=acl-fixed-case>OAQA</span> at <span class=acl-fixed-case>B</span>io<span class=acl-fixed-case>ASQ</span> 5<span class=acl-fixed-case>B</span></a></strong><br><a href=/people/k/khyathi-chandu/>Khyathi Chandu</a>
|
<a href=/people/a/aakanksha-naik/>Aakanksha Naik</a>
|
<a href=/people/a/aditya-chandrasekar/>Aditya Chandrasekar</a>
|
<a href=/people/z/zi-yang/>Zi Yang</a>
|
<a href=/people/n/niloy-gupta/>Niloy Gupta</a>
|
<a href=/people/e/eric-nyberg/>Eric Nyberg</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2307><div class="card-body p-3 small">In this paper, we describe our participation in phase B of task 5b of the fifth edition of the annual BioASQ challenge, which includes answering factoid, list, yes-no and summary questions from biomedical data. We describe our techniques with an emphasis on ideal answer generation, where the goal is to produce a relevant, precise, non-redundant, query-oriented summary from multiple relevant documents. We make use of extractive summarization techniques to address this task and experiment with different biomedical ontologies and various algorithms including <a href=https://en.wikipedia.org/wiki/Agglomerative_clustering>agglomerative clustering</a>, Maximum Marginal Relevance (MMR) and sentence compression. We propose a novel word embedding based tf-idf similarity metric and a soft positional constraint which improve our system performance. We evaluate our <a href=https://en.wikipedia.org/wiki/Method_(computer_programming)>techniques</a> on test batch 4 from the fourth edition of the challenge. Our best system achieves a ROUGE-2 score of 0.6534 and ROUGE-SU4 score of 0.6536.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2308.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2308 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2308 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2308/>Macquarie University at BioASQ 5b Query-based Summarisation Techniques for Selecting the Ideal Answers<span class=acl-fixed-case>M</span>acquarie <span class=acl-fixed-case>U</span>niversity at <span class=acl-fixed-case>B</span>io<span class=acl-fixed-case>ASQ</span> 5b – Query-based Summarisation Techniques for Selecting the Ideal Answers</a></strong><br><a href=/people/d/diego-molla/>Diego Mollá</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2308><div class="card-body p-3 small">Macquarie University&#8217;s contribution to the BioASQ challenge (Task 5b Phase B) focused on the use of query-based extractive summarisation techniques for the generation of the ideal answers. Four runs were submitted, with approaches ranging from a <a href=https://en.wikipedia.org/wiki/Triviality_(mathematics)>trivial system</a> that selected the first n snippets, to the use of <a href=https://en.wikipedia.org/wiki/Deep_learning>deep learning approaches</a> under a <a href=https://en.wikipedia.org/wiki/Regression_analysis>regression framework</a>. Our experiments and the ROUGE results of the five test batches of BioASQ indicate surprisingly good results for the trivial approach. Overall, most of our runs on the first three test batches achieved the best ROUGE-SU4 results in the challenge.<tex-math>n</tex-math> snippets, to the use of deep learning approaches under a regression framework. Our experiments and the ROUGE results of the five test batches of BioASQ indicate surprisingly good results for the trivial approach. Overall, most of our runs on the first three test batches achieved the best ROUGE-SU4 results in the challenge.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2309.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2309 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2309 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"><a class="badge badge-attachment align-middle mr-1" href=https://aclanthology.org/attachments/W17-2309.Presentation.pdf data-toggle=tooltip data-placement=top title=Presentation><i class="fas fa-file-powerpoint"></i></a></span>
<span class=d-block><strong><a class=align-middle href=/W17-2309/>Neural Question Answering at BioASQ 5B<span class=acl-fixed-case>B</span>io<span class=acl-fixed-case>ASQ</span> 5<span class=acl-fixed-case>B</span></a></strong><br><a href=/people/g/georg-wiese/>Georg Wiese</a>
|
<a href=/people/d/dirk-weissenborn/>Dirk Weissenborn</a>
|
<a href=/people/m/mariana-neves/>Mariana Neves</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2309><div class="card-body p-3 small">This paper describes our submission to the 2017 BioASQ challenge. We participated in Task B, Phase B which is concerned with biomedical question answering (QA). We focus on factoid and list question, using an extractive QA model, that is, we restrict our system to output substrings of the provided text snippets. At the core of our <a href=https://en.wikipedia.org/wiki/System>system</a>, we use FastQA, a state-of-the-art neural QA system. We extended it with biomedical word embeddings and changed its answer layer to be able to answer list questions in addition to factoid questions. We pre-trained the model on a large-scale open-domain QA dataset, SQuAD, and then fine-tuned the parameters on the BioASQ training set. With our approach, we achieve state-of-the-art results on factoid questions and competitive results on list questions.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2310.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2310 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2310 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"><a class="badge badge-secondary align-middle mr-1 pwc-reduced-padding" href="https://paperswithcode.com/paper/?acl=W17-2310" data-toggle=tooltip data-placement=top title=Code><svg xmlns="http://www.w3.org/2000/svg" class="pwc-icon-small" viewBox="0 0 512 512"><path stroke="#4d8093" fill="#4d8093" d="M88 128h48v256H88zm144 0h48v256h-48zm-72 16h48v224h-48zm144 0h48v224h-48zm72-16h48v256h-48z"/><path stroke="#4d8093" fill="#4d8093" d="M104 104V56H16v4e2h88v-48H64V104zM408 56v48h40v304h-40v48h88V56z"/></svg></a></span><span class=d-block><strong><a class=align-middle href=/W17-2310/>End-to-End System for Bacteria Habitat Extraction</a></strong><br><a href=/people/f/farrokh-mehryary/>Farrokh Mehryary</a>
|
<a href=/people/k/kai-hakala/>Kai Hakala</a>
|
<a href=/people/s/suwisa-kaewphan/>Suwisa Kaewphan</a>
|
<a href=/people/j/jari-bjorne/>Jari Björne</a>
|
<a href=/people/t/tapio-salakoski/>Tapio Salakoski</a>
|
<a href=/people/f/filip-ginter/>Filip Ginter</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2310><div class="card-body p-3 small">We introduce an end-to-end system capable of <a href=https://en.wikipedia.org/wiki/Named-entity_recognition>named-entity detection</a>, <a href=https://en.wikipedia.org/wiki/Normalization_(statistics)>normalization</a> and relation extraction for extracting information about bacteria and their habitats from <a href=https://en.wikipedia.org/wiki/Medical_literature>biomedical literature</a>. Our system is based on <a href=https://en.wikipedia.org/wiki/Deep_learning>deep learning</a>, CRF classifiers and <a href=https://en.wikipedia.org/wiki/Vector_space_model>vector space models</a>. We train and evaluate the <a href=https://en.wikipedia.org/wiki/System>system</a> on the BioNLP 2016 Shared Task Bacteria Biotope data. The official evaluation shows that the joint performance of our entity detection and relation extraction models outperforms the winning team of the Shared Task by 19pp on F1-score, establishing a new top score for the task. We also achieve state-of-the-art results in the <a href=https://en.wikipedia.org/wiki/Normalization_(statistics)>normalization task</a>. Our <a href=https://en.wikipedia.org/wiki/System>system</a> is open source and freely available at.<url>https://github.com/TurkuNLP/BHE</url>.\n</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2312.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2312 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2312 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2312/>Representation of complex terms in a <a href=https://en.wikipedia.org/wiki/Vector_space>vector space</a> structured by an <a href=https://en.wikipedia.org/wiki/Ontology_(information_science)>ontology</a> for a normalization task</a></strong><br><a href=/people/a/arnaud-ferre/>Arnaud Ferré</a>
|
<a href=/people/p/pierre-zweigenbaum/>Pierre Zweigenbaum</a>
|
<a href=/people/c/claire-nedellec/>Claire Nédellec</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2312><div class="card-body p-3 small">We propose in this paper a <a href=https://en.wikipedia.org/wiki/Semi-supervised_learning>semi-supervised method</a> for labeling terms of texts with concepts of a <a href=https://en.wikipedia.org/wiki/Ontology_(information_science)>domain ontology</a>. The method generates continuous vector representations of complex terms in a <a href=https://en.wikipedia.org/wiki/Semantic_space>semantic space</a> structured by the <a href=https://en.wikipedia.org/wiki/Ontology_(information_science)>ontology</a>. The proposed method relies on a distributional semantics approach, which generates initial vectors for each of the extracted terms. Then these <a href=https://en.wikipedia.org/wiki/Vector_(mathematics_and_physics)>vectors</a> are embedded in the <a href=https://en.wikipedia.org/wiki/Vector_space>vector space</a> constructed from the structure of the ontology. This embedding is carried out by training a <a href=https://en.wikipedia.org/wiki/Linear_model>linear model</a>. Finally, we apply a distance calculation to determine the proximity between vectors of terms and vectors of concepts and thus to assign <a href=https://en.wikipedia.org/wiki/Ontology_(information_science)>ontology labels</a> to terms. We have evaluated the quality of these representations for a normalization task by using the concepts of an <a href=https://en.wikipedia.org/wiki/Ontology_(information_science)>ontology</a> as semantic labels. Normalization of terms is an important step to extract a part of the information containing in texts, but the <a href=https://en.wikipedia.org/wiki/Vector_space>vector space</a> generated might find other applications. The performance of this <a href=https://en.wikipedia.org/wiki/Methodology>method</a> is comparable to that of the state of the art for this task of standardization, opening up encouraging prospects.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2313.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2313 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2313 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2313/>Improving Correlation with Human Judgments by Integrating Semantic Similarity with SecondOrder Vectors</a></strong><br><a href=/people/b/bridget-mcinnes/>Bridget McInnes</a>
|
<a href=/people/t/ted-pedersen/>Ted Pedersen</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2313><div class="card-body p-3 small">Vector space methods that measure semantic similarity and relatedness often rely on distributional information such as cooccurrence frequencies or <a href=https://en.wikipedia.org/wiki/Correlation_and_dependence>statistical measures of association</a> to weight the importance of particular cooccurrences. In this paper, we extend these methods by incorporating a measure of <a href=https://en.wikipedia.org/wiki/Semantic_similarity>semantic similarity</a> based on a human curated taxonomy into a secondorder vector representation. This results in a measure of <a href=https://en.wikipedia.org/wiki/Semantic_similarity>semantic relatedness</a> that combines both the <a href=https://en.wikipedia.org/wiki/Context_(language_use)>contextual information</a> available in a corpusbased vector space representation with the semantic knowledge found in a biomedical ontology. Our results show that incorporating <a href=https://en.wikipedia.org/wiki/Semantic_similarity>semantic similarity</a> into a second order co-occurrence matrices improves correlation with human judgments for both similarity and relatedness, and that our method compares favorably to various different word embedding methods that have recently been evaluated on the same reference standards we have used.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2314.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2314 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2314 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2314/>Proactive Learning for Named Entity Recognition</a></strong><br><a href=/people/m/maolin-li/>Maolin Li</a>
|
<a href=/people/n/nhung-nguyen/>Nhung Nguyen</a>
|
<a href=/people/s/sophia-ananiadou/>Sophia Ananiadou</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2314><div class="card-body p-3 small">The goal of <a href=https://en.wikipedia.org/wiki/Active_learning>active learning</a> is to minimise the cost of producing an <a href=https://en.wikipedia.org/wiki/Annotation>annotated dataset</a>, in which annotators are assumed to be perfect, i.e., they always choose the correct labels. However, in practice, annotators are not infallible, and they are likely to assign incorrect labels to some instances. Proactive learning is a generalisation of active learning that can model different kinds of <a href=https://en.wikipedia.org/wiki/Annotation>annotators</a>. Although <a href=https://en.wikipedia.org/wiki/Proactive_learning>proactive learning</a> has been applied to certain labelling tasks, such as text classification, there is little work on its application to named entity (NE) tagging. In this paper, we propose a proactive learning method for producing NE annotated corpora, using two annotators with different levels of expertise, and who charge different amounts based on their levels of experience. To optimise both cost and annotation quality, we also propose a mechanism to present multiple sentences to annotators at each iteration. Experimental results for several corpora show that our method facilitates the construction of high-quality NE labelled datasets at minimal cost.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2315.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2315 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2315 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2315/>Biomedical Event Extraction using Abstract Meaning Representation<span class=acl-fixed-case>A</span>bstract <span class=acl-fixed-case>M</span>eaning <span class=acl-fixed-case>R</span>epresentation</a></strong><br><a href=/people/s/sudha-rao/>Sudha Rao</a>
|
<a href=/people/d/daniel-marcu/>Daniel Marcu</a>
|
<a href=/people/k/kevin-knight/>Kevin Knight</a>
|
<a href=/people/h/hal-daume-iii/>Hal Daumé III</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2315><div class="card-body p-3 small">We propose a novel, Abstract Meaning Representation (AMR) based approach to identifying molecular events / interactions in biomedical text. Our key contributions are : (1) an empirical validation of our hypothesis that an event is a subgraph of the AMR graph, (2) a neural network-based model that identifies such an event subgraph given an AMR, and (3) a distant supervision based approach to gather additional training data. We evaluate our approach on the 2013 Genia Event Extraction dataset and show promising results.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2316.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2316 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2316 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2316/>Detecting Personal Medication Intake in <a href=https://en.wikipedia.org/wiki/Twitter>Twitter</a> : An Annotated Corpus and Baseline Classification System<span class=acl-fixed-case>T</span>witter: An Annotated Corpus and Baseline Classification System</a></strong><br><a href=/people/a/ari-klein/>Ari Klein</a>
|
<a href=/people/a/abeed-sarker/>Abeed Sarker</a>
|
<a href=/people/m/masoud-rouhizadeh/>Masoud Rouhizadeh</a>
|
<a href=/people/k/karen-oconnor/>Karen O’Connor</a>
|
<a href=/people/g/graciela-gonzalez/>Graciela Gonzalez</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2316><div class="card-body p-3 small">Social media sites (e.g., <a href=https://en.wikipedia.org/wiki/Twitter>Twitter</a>) have been used for surveillance of drug safety at the population level, but studies that focus on the effects of medications on specific sets of individuals have had to rely on other sources of data. Mining social media data for this in-formation would require the ability to distinguish indications of personal medication in-take in this <a href=https://en.wikipedia.org/wiki/Media_(communication)>media</a>. Towards that end, this paper presents an annotated corpus that can be used to train <a href=https://en.wikipedia.org/wiki/Machine_learning>machine learning systems</a> to determine whether a tweet that mentions a medication indicates that the individual posting has taken that medication at a specific time. To demonstrate the utility of the <a href=https://en.wikipedia.org/wiki/Text_corpus>corpus</a> as a training set, we present baseline results of <a href=https://en.wikipedia.org/wiki/Supervised_classification>supervised classification</a>.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2317.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2317 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2317 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2317/>Unsupervised Context-Sensitive Spelling Correction of Clinical Free-Text with Word and Character N-Gram Embeddings</a></strong><br><a href=/people/p/pieter-fivez/>Pieter Fivez</a>
|
<a href=/people/s/simon-suster/>Simon Šuster</a>
|
<a href=/people/w/walter-daelemans/>Walter Daelemans</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2317><div class="card-body p-3 small">We present an unsupervised context-sensitive spelling correction method for clinical free-text that uses word and character n-gram embeddings. Our method generates misspelling replacement candidates and ranks them according to their semantic fit, by calculating a weighted cosine similarity between the vectorized representation of a candidate and the misspelling context. We greatly outperform two baseline off-the-shelf spelling correction tools on a manually annotated MIMIC-III test set, and counter the frequency bias of an optimized noisy channel model, showing that neural embeddings can be successfully exploited to include context-awareness in a spelling correction model.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2318.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2318 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2318 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2318/>Characterization of Divergence in Impaired Speech of ALS Patients<span class=acl-fixed-case>ALS</span> Patients</a></strong><br><a href=/people/a/archna-bhatia/>Archna Bhatia</a>
|
<a href=/people/b/bonnie-dorr/>Bonnie Dorr</a>
|
<a href=/people/k/kristy-hollingshead/>Kristy Hollingshead</a>
|
<a href=/people/s/samuel-l-phillips/>Samuel L. Phillips</a>
|
<a href=/people/b/barbara-mckenzie/>Barbara McKenzie</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2318><div class="card-body p-3 small">Approximately 80 % to 95 % of patients with Amyotrophic Lateral Sclerosis (ALS) eventually develop speech impairments, such as defective articulation, slow laborious speech and <a href=https://en.wikipedia.org/wiki/Hypernasal_speech>hypernasality</a>. The relationship between <a href=https://en.wikipedia.org/wiki/Speech_disorder>impaired speech</a> and asymptomatic speech may be seen as a divergence from a baseline. This relationship can be characterized in terms of measurable combinations of <a href=https://en.wikipedia.org/wiki/Phonology>phonological characteristics</a> that are indicative of the degree to which the two diverge. We demonstrate that divergence measurements based on <a href=https://en.wikipedia.org/wiki/Phonology>phonological characteristics of speech</a> correlate with <a href=https://en.wikipedia.org/wiki/Physiology>physiological assessments of ALS</a>. Speech-based assessments offer benefits over commonly-used physiological assessments in that they are inexpensive, non-intrusive, and do not require trained clinical personnel for administering and interpreting the results.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2319.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2319 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2319 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2319/>Deep Learning for Punctuation Restoration in Medical Reports</a></strong><br><a href=/people/w/wael-salloum/>Wael Salloum</a>
|
<a href=/people/g/gregory-finley/>Greg Finley</a>
|
<a href=/people/e/erik-edwards/>Erik Edwards</a>
|
<a href=/people/m/mark-miller/>Mark Miller</a>
|
<a href=/people/d/david-suendermann-oeft/>David Suendermann-Oeft</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2319><div class="card-body p-3 small">In clinical dictation, speakers try to be as concise as possible to save time, often resulting in utterances without explicit punctuation commands. Since the end product of a dictated report, e.g. an out-patient letter, does require correct <a href=https://en.wikipedia.org/wiki/Orthography>orthography</a>, including exact <a href=https://en.wikipedia.org/wiki/Punctuation>punctuation</a>, the latter need to be restored, preferably by automated means. This paper describes a method for punctuation restoration based on a state-of-the-art stack of NLP and machine learning techniques including B-RNNs with an attention mechanism and late fusion, as well as a feature extraction technique tailored to the processing of medical terminology using a novel vocabulary reduction model. To the best of our knowledge, the resulting performance is superior to that reported in prior art on similar <a href=https://en.wikipedia.org/wiki/Task_(project_management)>tasks</a>.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2320.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2320 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2320 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2320/>Unsupervised Domain Adaptation for Clinical Negation Detection</a></strong><br><a href=/people/t/timothy-miller/>Timothy Miller</a>
|
<a href=/people/s/steven-bethard/>Steven Bethard</a>
|
<a href=/people/h/hadi-amiri/>Hadi Amiri</a>
|
<a href=/people/g/guergana-savova/>Guergana Savova</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2320><div class="card-body p-3 small">Detecting negated concepts in clinical texts is an important part of NLP information extraction systems. However, generalizability of negation systems is lacking, as cross-domain experiments suffer dramatic performance losses. We examine the performance of multiple unsupervised domain adaptation algorithms on clinical negation detection, finding only modest gains that fall well short of in-domain performance.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2321.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2321 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2321 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2321/>BioCreative VI Precision Medicine Track : creating a training corpus for mining protein-protein interactions affected by mutations<span class=acl-fixed-case>B</span>io<span class=acl-fixed-case>C</span>reative <span class=acl-fixed-case>VI</span> Precision Medicine Track: creating a training corpus for mining protein-protein interactions affected by mutations</a></strong><br><a href=/people/r/rezarta-islamaj-dogan/>Rezarta Islamaj Doğan</a>
|
<a href=/people/a/andrew-chatr-aryamontri/>Andrew Chatr-aryamontri</a>
|
<a href=/people/s/sun-kim/>Sun Kim</a>
|
<a href=/people/c/chih-hsuan-wei/>Chih-Hsuan Wei</a>
|
<a href=/people/y/yifan-peng/>Yifan Peng</a>
|
<a href=/people/d/donald-c-comeau/>Donald Comeau</a>
|
<a href=/people/z/zhiyong-lu/>Zhiyong Lu</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2321><div class="card-body p-3 small">The Precision Medicine Track in BioCre-ative VI aims to bring together the Bi-oNLP community for a novel challenge focused on mining the biomedical litera-ture in search of mutations and protein-protein interactions (PPI). In order to support this track with an effective train-ing dataset with limited curator time, the track organizers carefully reviewed Pub-Med articles from two different sources : curated public PPI databases, and the re-sults of state-of-the-art public text mining tools. We detail here the <a href=https://en.wikipedia.org/wiki/Data_collection>data collection</a>, manual review and annotation process and describe this training corpus charac-teristics. We also describe a corpus per-formance baseline. This analysis will provide useful information to developers and researchers for comparing and devel-oping innovative text mining approaches for the BioCreative VI challenge and other Precision Medicine related applica-tions.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2322.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2322 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2322 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2322/>Painless Relation Extraction with Kindred</a></strong><br><a href=/people/j/jake-lever/>Jake Lever</a>
|
<a href=/people/s/steven-jm-jones/>Steven Jones</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2322><div class="card-body p-3 small">Relation extraction methods are essential for creating robust text mining tools to help researchers find useful knowledge in the vast published literature. Easy-to-use and generalizable methods are needed to encourage an ecosystem in which researchers can easily use shared resources and build upon each others&#8217; methods. We present the Kindred Python package for <a href=https://en.wikipedia.org/wiki/Relation_extraction>relation extraction</a>. It builds upon methods from the most successful tools in the recent BioNLP Shared Task to predict high-quality predictions with low computational cost. It also integrates with PubAnnotation, PubTator, and BioNLP Shared Task data in order to allow easy development and application of relation extraction models.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2323.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2323 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2323 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2323/>Noise Reduction Methods for Distantly Supervised Biomedical Relation Extraction</a></strong><br><a href=/people/g/gang-li/>Gang Li</a>
|
<a href=/people/c/cathy-wu/>Cathy Wu</a>
|
<a href=/people/k/k-vijay-shanker/>K. Vijay-Shanker</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2323><div class="card-body p-3 small">Distant supervision has been applied to automatically generate labeled data for biomedical relation extraction. Noise exists in both positively and negatively-labeled data and affects the performance of supervised machine learning methods. In this paper, we propose three novel <a href=https://en.wikipedia.org/wiki/Heuristic>heuristics</a> based on the notion of proximity, trigger word and confidence of patterns to leverage lexical and syntactic information to reduce the level of noise in the distantly labeled data. Experiments on three different tasks, extraction of protein-protein-interaction, miRNA-gene regulation relation and protein-localization event, show that the proposed methods can improve the <a href=https://en.wikipedia.org/wiki/F-score>F-score</a> over the baseline by 6, 10 and 14 points for the three tasks, respectively. We also show that when the <a href=https://en.wikipedia.org/wiki/Mathematical_model>models</a> are configured to output high-confidence results, high precisions can be obtained using the proposed methods, making them promising for facilitating manual curation for <a href=https://en.wikipedia.org/wiki/Database>databases</a>.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2324.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2324 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2324 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2324/>Role-Preserving Redaction of Medical Records to Enable Ontology-Driven Processing</a></strong><br><a href=/people/s/seth-polsley/>Seth Polsley</a>
|
<a href=/people/a/atif-tahir/>Atif Tahir</a>
|
<a href=/people/m/muppala-raju/>Muppala Raju</a>
|
<a href=/people/a/akintayo-akinleye/>Akintayo Akinleye</a>
|
<a href=/people/d/duane-steward/>Duane Steward</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2324><div class="card-body p-3 small">Electronic medical records (EMR) have largely replaced hand-written patient files in healthcare. The growing pool of <a href=https://en.wikipedia.org/wiki/Electronic_health_record>EMR data</a> presents a significant resource in <a href=https://en.wikipedia.org/wiki/Medical_research>medical research</a>, but the U.S. Health Insurance Portability and Accountability Act (HIPAA) mandates redacting <a href=https://en.wikipedia.org/wiki/Medical_record>medical records</a> before performing any analysis on the same. This process complicates obtaining <a href=https://en.wikipedia.org/wiki/Medical_record>medical data</a> and can remove much useful information from the record. As part of a larger project involving ontology-driven medical processing, we employ a method of recognizing protected health information (PHI) that maps to <a href=https://en.wikipedia.org/wiki/Ontology_(information_science)>ontological terms</a>. We then use the relationships defined in the <a href=https://en.wikipedia.org/wiki/Ontology_(information_science)>ontology</a> to redact medical texts so that roles and semantics of terms are retained without compromising <a href=https://en.wikipedia.org/wiki/Anonymity>anonymity</a>. The method is evaluated by clinical experts on several hundred <a href=https://en.wikipedia.org/wiki/Medical_record>medical documents</a>, achieving up to a 98.8 % <a href=https://en.wikipedia.org/wiki/F-score>f-score</a>, and has already shown promise for retaining semantic information in later processing.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2325.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2325 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2325 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2325/>Annotation of pain and anesthesia events for surgery-related processes and outcomes extraction</a></strong><br><a href=/people/w/wen-wai-yim/>Wen-wai Yim</a>
|
<a href=/people/d/dario-tedesco/>Dario Tedesco</a>
|
<a href=/people/c/catherine-curtin/>Catherine Curtin</a>
|
<a href=/people/t/tina-hernandez-boussard/>Tina Hernandez-Boussard</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2325><div class="card-body p-3 small">Pain and anesthesia information are crucial elements to identifying surgery-related processes and outcomes. However <a href=https://en.wikipedia.org/wiki/Pain>pain</a> is not consistently recorded in the <a href=https://en.wikipedia.org/wiki/Electronic_health_record>electronic medical record</a>. Even when recorded, the rich complex granularity of the pain experience may be lost. Similarly, anesthesia information is recorded using local electronic collection systems ; though the <a href=https://en.wikipedia.org/wiki/Accuracy_and_precision>accuracy</a> and completeness of the information is unknown. We propose an annotation schema to capture <a href=https://en.wikipedia.org/wiki/Pain>pain</a>, <a href=https://en.wikipedia.org/wiki/Pain_management>pain management</a>, and anesthesia event information.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2326.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2326 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2326 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2326/>Identifying Comparative Structures in Biomedical Text</a></strong><br><a href=/people/s/samir-gupta/>Samir Gupta</a>
|
<a href=/people/a/a-s-m-ashique-mahmood/>A.S.M. Ashique Mahmood</a>
|
<a href=/people/k/karen-ross/>Karen Ross</a>
|
<a href=/people/c/cathy-wu/>Cathy Wu</a>
|
<a href=/people/k/k-vijay-shanker/>K. Vijay-Shanker</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2326><div class="card-body p-3 small">Comparison sentences are very commonly used by authors in <a href=https://en.wikipedia.org/wiki/Medical_literature>biomedical literature</a> to report results of experiments. In such comparisons, authors typically make observations under two different scenarios. In this paper, we present a <a href=https://en.wikipedia.org/wiki/System>system</a> to automatically identify such comparative sentences and their components i.e. the compared entities, the scale of the comparison and the aspect on which the entities are being compared. Our <a href=https://en.wikipedia.org/wiki/Methodology>methodology</a> is based on <a href=https://en.wikipedia.org/wiki/Coupling_(computer_programming)>dependencies</a> obtained by applying a <a href=https://en.wikipedia.org/wiki/Parsing>parser</a> to extract a wide range of comparison structures. We evaluated our <a href=https://en.wikipedia.org/wiki/System>system</a> for its effectiveness in identifying comparisons and their components. The <a href=https://en.wikipedia.org/wiki/System>system</a> achieved a F-score of 0.87 for comparison sentence identification and 0.77-0.81 for identifying its components.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2327.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2327 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2327 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2327/>Tagging Funding Agencies and Grants in Scientific Articles using Sequential Learning Models</a></strong><br><a href=/people/s/subhradeep-kayal/>Subhradeep Kayal</a>
|
<a href=/people/z/zubair-afzal/>Zubair Afzal</a>
|
<a href=/people/g/george-tsatsaronis/>George Tsatsaronis</a>
|
<a href=/people/s/sophia-katrenko/>Sophia Katrenko</a>
|
<a href=/people/p/pascal-coupet/>Pascal Coupet</a>
|
<a href=/people/m/marius-doornenbal/>Marius Doornenbal</a>
|
<a href=/people/m/michelle-gregory/>Michelle Gregory</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2327><div class="card-body p-3 small">In this paper we present a solution for tagging funding bodies and grants in scientific articles using a combination of trained sequential learning models, namely conditional random fields (CRF), hidden markov models (HMM) and maximum entropy models (MaxEnt), on a benchmark set created in-house. We apply the trained models to address the BioASQ challenge 5c, which is a newly introduced task that aims to solve the problem of funding information extraction from scientific articles. Results in the dry-run data set of BioASQ task 5c show that the suggested approach can achieve a micro-recall of more than 85 % in tagging both funding bodies and grants.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2328.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2328 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2328 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2328/>Deep Learning for Biomedical Information Retrieval : Learning Textual Relevance from Click Logs</a></strong><br><a href=/people/s/sunil-mohan/>Sunil Mohan</a>
|
<a href=/people/n/nicolas-fiorini/>Nicolas Fiorini</a>
|
<a href=/people/s/sun-kim/>Sun Kim</a>
|
<a href=/people/z/zhiyong-lu/>Zhiyong Lu</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2328><div class="card-body p-3 small">We describe a Deep Learning approach to modeling the relevance of a document&#8217;s text to a query, applied to biomedical literature. Instead of mapping each document and query to a common semantic space, we compute a variable-length difference vector between the query and document which is then passed through a deep convolution stage followed by a deep regression network to produce the estimated probability of the document&#8217;s relevance to the query. Despite the small amount of training data, this approach produces a more robust predictor than computing similarities between semantic vector representations of the query and document, and also results in significant improvements over traditional IR text factors. In the future, we plan to explore its application in improving <a href=https://en.wikipedia.org/wiki/PubMed>PubMed search</a>.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2329.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2329 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2329 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"><a class="badge badge-secondary align-middle mr-1 pwc-reduced-padding" href="https://paperswithcode.com/paper/?acl=W17-2329" data-toggle=tooltip data-placement=top title=Code><svg xmlns="http://www.w3.org/2000/svg" class="pwc-icon-small" viewBox="0 0 512 512"><path stroke="#4d8093" fill="#4d8093" d="M88 128h48v256H88zm144 0h48v256h-48zm-72 16h48v224h-48zm144 0h48v224h-48zm72-16h48v256h-48z"/><path stroke="#4d8093" fill="#4d8093" d="M104 104V56H16v4e2h88v-48H64V104zM408 56v48h40v304h-40v48h88V56z"/></svg></a></span><span class=d-block><strong><a class=align-middle href=/W17-2329/>Detecting Dementia through Retrospective Analysis of Routine Blog Posts by Bloggers with Dementia</a></strong><br><a href=/people/v/vaden-masrani/>Vaden Masrani</a>
|
<a href=/people/g/gabriel-murray/>Gabriel Murray</a>
|
<a href=/people/t/thalia-field/>Thalia Field</a>
|
<a href=/people/g/giuseppe-carenini/>Giuseppe Carenini</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2329><div class="card-body p-3 small">We investigate if writers with dementia can be automatically distinguished from those without by analyzing <a href=https://en.wikipedia.org/wiki/Marker_(linguistics)>linguistic markers</a> in <a href=https://en.wikipedia.org/wiki/Writing>written text</a>, in the form of <a href=https://en.wikipedia.org/wiki/Blog>blog posts</a>. We have built a <a href=https://en.wikipedia.org/wiki/Text_corpus>corpus</a> of several thousand <a href=https://en.wikipedia.org/wiki/Blog>blog posts</a>, some by people with dementia and others by people with loved ones with dementia. We use this <a href=https://en.wikipedia.org/wiki/Data_set>dataset</a> to train and test several <a href=https://en.wikipedia.org/wiki/Machine_learning>machine learning methods</a>, and achieve <a href=https://en.wikipedia.org/wiki/Prediction>prediction</a> performance at a level far above the <a href=https://en.wikipedia.org/wiki/Baseline_(configuration_management)>baseline</a>.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2330.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2330 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2330 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2330/>Protein Word Detection using Text Segmentation Techniques</a></strong><br><a href=/people/d/devi-ganesan/>Devi Ganesan</a>
|
<a href=/people/a/ashish-v-tendulkar/>Ashish V. Tendulkar</a>
|
<a href=/people/s/sutanu-chakraborti/>Sutanu Chakraborti</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2330><div class="card-body p-3 small">Literature in <a href=https://en.wikipedia.org/wiki/Molecular_biology>Molecular Biology</a> is abundant with linguistic metaphors. There have been works in the past that attempt to draw parallels between <a href=https://en.wikipedia.org/wiki/Linguistics>linguistics</a> and <a href=https://en.wikipedia.org/wiki/Biology>biology</a>, driven by the fundamental premise that <a href=https://en.wikipedia.org/wiki/Protein>proteins</a> have a language of their own. Since word detection is crucial to the decipherment of any unknown language, we attempt to establish a problem mapping from <a href=https://en.wikipedia.org/wiki/Natural_language>natural language text</a> to <a href=https://en.wikipedia.org/wiki/Protein_primary_structure>protein sequences</a> at the level of words. Towards this end, we explore the use of an unsupervised text segmentation algorithm to the task of extracting biological words from <a href=https://en.wikipedia.org/wiki/Protein_primary_structure>protein sequences</a>. In particular, we demonstrate the effectiveness of using <a href=https://en.wikipedia.org/wiki/Domain_knowledge>domain knowledge</a> to complement data driven approaches in the text segmentation task, as well as in its biological counterpart. We also propose a novel extrinsic evaluation measure for protein words through protein family classification.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2331.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2331 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2331 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"><a class="badge badge-secondary align-middle mr-1 pwc-reduced-padding" href="https://paperswithcode.com/paper/?acl=W17-2331" data-toggle=tooltip data-placement=top title=Code><svg xmlns="http://www.w3.org/2000/svg" class="pwc-icon-small" viewBox="0 0 512 512"><path stroke="#4d8093" fill="#4d8093" d="M88 128h48v256H88zm144 0h48v256h-48zm-72 16h48v224h-48zm144 0h48v224h-48zm72-16h48v256h-48z"/><path stroke="#4d8093" fill="#4d8093" d="M104 104V56H16v4e2h88v-48H64V104zM408 56v48h40v304h-40v48h88V56z"/></svg></a></span><span class=d-block><strong><a class=align-middle href=/W17-2331/>External Evaluation of Event Extraction Classifiers for Automatic Pathway Curation : An extended study of the <a href=https://en.wikipedia.org/wiki/MTOR_pathway>mTOR pathway</a><span class=acl-fixed-case>TOR</span> pathway</a></strong><br><a href=/people/w/wojciech-kusa/>Wojciech Kusa</a>
|
<a href=/people/m/michael-spranger/>Michael Spranger</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2331><div class="card-body p-3 small">This paper evaluates the impact of various event extraction systems on automatic pathway curation using the popular <a href=https://en.wikipedia.org/wiki/MTOR_pathway>mTOR pathway</a>. We quantify the impact of <a href=https://en.wikipedia.org/wiki/Training,_validation,_and_test_sets>training data sets</a> as well as different <a href=https://en.wikipedia.org/wiki/Statistical_classification>machine learning classifiers</a> and show that some improve the quality of automatically extracted pathways.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2332.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2332 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2332 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"><a class="badge badge-secondary align-middle mr-1 pwc-reduced-padding" href="https://paperswithcode.com/paper/?acl=W17-2332" data-toggle=tooltip data-placement=top title=Code><svg xmlns="http://www.w3.org/2000/svg" class="pwc-icon-small" viewBox="0 0 512 512"><path stroke="#4d8093" fill="#4d8093" d="M88 128h48v256H88zm144 0h48v256h-48zm-72 16h48v224h-48zm144 0h48v224h-48zm72-16h48v256h-48z"/><path stroke="#4d8093" fill="#4d8093" d="M104 104V56H16v4e2h88v-48H64V104zM408 56v48h40v304h-40v48h88V56z"/></svg></a></span><span class=d-block><strong><a class=align-middle href=/W17-2332/>Toward Automated Early Sepsis Alerting : Identifying Infection Patients from Nursing Notes</a></strong><br><a href=/people/e/emilia-apostolova/>Emilia Apostolova</a>
|
<a href=/people/t/tom-velez/>Tom Velez</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2332><div class="card-body p-3 small">Severe sepsis and <a href=https://en.wikipedia.org/wiki/Septic_shock>septic shock</a> are conditions that affect millions of patients and have close to 50 % <a href=https://en.wikipedia.org/wiki/Mortality_rate>mortality rate</a>. Early identification of at-risk patients significantly improves outcomes. Electronic surveillance tools have been developed to monitor structured Electronic Medical Records and automatically recognize early signs of sepsis. However, many sepsis risk factors (e.g. symptoms and signs of infection) are often captured only in free text clinical notes. In this study, we developed a method for automatic monitoring of nursing notes for signs and symptoms of infection. We utilized a creative approach to automatically generate an annotated dataset. The <a href=https://en.wikipedia.org/wiki/Data_set>dataset</a> was used to create a <a href=https://en.wikipedia.org/wiki/Machine_learning>Machine Learning model</a> that achieved an <a href=https://en.wikipedia.org/wiki/F-number>F1-score</a> ranging from 79 to 96 %.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2333.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2333 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2333 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2333/>Enhancing Automatic ICD-9-CM Code Assignment for Medical Texts with PubMed<span class=acl-fixed-case>ICD</span>-9-<span class=acl-fixed-case>CM</span> Code Assignment for Medical Texts with <span class=acl-fixed-case>P</span>ub<span class=acl-fixed-case>M</span>ed</a></strong><br><a href=/people/d/danchen-zhang/>Danchen Zhang</a>
|
<a href=/people/d/daqing-he/>Daqing He</a>
|
<a href=/people/s/sanqiang-zhao/>Sanqiang Zhao</a>
|
<a href=/people/l/lei-li/>Lei Li</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2333><div class="card-body p-3 small">Assigning a standard ICD-9-CM code to <a href=https://en.wikipedia.org/wiki/Symptom>disease symptoms</a> in <a href=https://en.wikipedia.org/wiki/Medical_literature>medical texts</a> is an important task in the <a href=https://en.wikipedia.org/wiki/Medicine>medical domain</a>. Automating this <a href=https://en.wikipedia.org/wiki/Process_(engineering)>process</a> could greatly reduce the costs. However, the effectiveness of an automatic ICD-9-CM code classifier faces a serious problem, which can be triggered by <a href=https://en.wikipedia.org/wiki/Training,_validation,_and_test_sets>unbalanced training data</a>. Frequent diseases often have more <a href=https://en.wikipedia.org/wiki/Training,_validation,_and_test_sets>training data</a>, which helps its <a href=https://en.wikipedia.org/wiki/Medical_classification>classification</a> to perform better than that of an infrequent disease. However, a <a href=https://en.wikipedia.org/wiki/Incidence_(epidemiology)>disease&#8217;s frequency</a> does not necessarily reflect its importance. To resolve this training data shortage problem, we propose to strategically draw data from <a href=https://en.wikipedia.org/wiki/PubMed>PubMed</a> to enrich the training data when there is such need. We validate our method on the CMC dataset, and the evaluation results indicate that our method can significantly improve the code assignment classifiers&#8217; performance at the macro-averaging level.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2334.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2334 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2334 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2334/>Evaluating Feature Extraction Methods for Knowledge-based Biomedical Word Sense Disambiguation</a></strong><br><a href=/people/s/sam-henry/>Sam Henry</a>
|
<a href=/people/c/clint-cuffy/>Clint Cuffy</a>
|
<a href=/people/b/bridget-mcinnes/>Bridget McInnes</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2334><div class="card-body p-3 small">In this paper, we present an analysis of <a href=https://en.wikipedia.org/wiki/Feature_extraction>feature extraction methods</a> via <a href=https://en.wikipedia.org/wiki/Dimensionality_reduction>dimensionality reduction</a> for the task of biomedical Word Sense Disambiguation (WSD). We modify the vector representations in the 2-MRD WSD algorithm, and evaluate four dimensionality reduction methods : Word Embeddings using Continuous Bag of Words and Skip Gram, Singular Value Decomposition (SVD), and Principal Component Analysis (PCA). We also evaluate the effects of <a href=https://en.wikipedia.org/wiki/Vector_space>vector size</a> on the performance of each of these <a href=https://en.wikipedia.org/wiki/Method_(computer_programming)>methods</a>. Results are evaluated on five standard evaluation datasets (Abbrev.100, Abbrev.200, Abbrev.300, NLM-WSD, and MSH-WSD). We find that <a href=https://en.wikipedia.org/wiki/Norm_(mathematics)>vector sizes</a> of 100 are sufficient for all techniques except <a href=https://en.wikipedia.org/wiki/Singular_value_decomposition>SVD</a>, for which a <a href=https://en.wikipedia.org/wiki/Vector_space>vector size</a> of 1500 is referred. We also show that <a href=https://en.wikipedia.org/wiki/Word_embedding>SVD</a> performs on par with <a href=https://en.wikipedia.org/wiki/Word_embedding>Word Embeddings</a> for all but one <a href=https://en.wikipedia.org/wiki/Data_set>dataset</a>.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2336.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2336 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2336 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2336/>Automated Preamble Detection in Dictated Medical Reports</a></strong><br><a href=/people/w/wael-salloum/>Wael Salloum</a>
|
<a href=/people/g/gregory-finley/>Greg Finley</a>
|
<a href=/people/e/erik-edwards/>Erik Edwards</a>
|
<a href=/people/m/mark-miller/>Mark Miller</a>
|
<a href=/people/d/david-suendermann-oeft/>David Suendermann-Oeft</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2336><div class="card-body p-3 small">Dictated medical reports very often feature a <a href=https://en.wikipedia.org/wiki/Preamble>preamble</a> containing <a href=https://en.wikipedia.org/wiki/Metainformation>metainformation</a> about the report such as patient and physician names, location and name of the clinic, date of procedure, and so on. In the medical transcription process, the <a href=https://en.wikipedia.org/wiki/Preamble>preamble</a> is usually omitted from the final report, as it contains information already available in the <a href=https://en.wikipedia.org/wiki/Electronic_health_record>electronic medical record</a>. We present a <a href=https://en.wikipedia.org/wiki/Scientific_method>method</a> which is able to automatically identify <a href=https://en.wikipedia.org/wiki/Preamble>preambles</a> in <a href=https://en.wikipedia.org/wiki/Dictation_(exercise)>medical dictations</a>. The method makes use of state-of-the-art NLP techniques including <a href=https://en.wikipedia.org/wiki/Word_embedding>word embeddings</a> and Bi-LSTMs and achieves preamble detection performance superior to humans.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2337.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2337 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2337 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2337/>A Biomedical Question Answering System in BioASQ 2017<span class=acl-fixed-case>B</span>io<span class=acl-fixed-case>ASQ</span> 2017</a></strong><br><a href=/people/m/mourad-sarrouti/>Mourad Sarrouti</a>
|
<a href=/people/s/said-ouatik-el-alaoui/>Said Ouatik El Alaoui</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2337><div class="card-body p-3 small">Question answering, the identification of short accurate answers to users questions, is a longstanding challenge widely studied over the last decades in the <a href=https://en.wikipedia.org/wiki/Open_domain>open domain</a>. However, <a href=https://en.wikipedia.org/wiki/Information_technology>it</a> still requires further efforts in the <a href=https://en.wikipedia.org/wiki/Biomedicine>biomedical domain</a>. In this paper, we describe our participation in phase B of task 5b in the 2017 BioASQ challenge using our biomedical question answering system. Our <a href=https://en.wikipedia.org/wiki/System>system</a>, dealing with four types of <a href=https://en.wikipedia.org/wiki/Question>questions</a> (i.e., yes / no, <a href=https://en.wikipedia.org/wiki/Factoid>factoid</a>, list, and summary), is based on (1) a dictionary-based approach for generating the exact answers of yes / no questions, (2) UMLS metathesaurus and term frequency metric for extracting the exact answers of <a href=https://en.wikipedia.org/wiki/Factoid>factoid and list questions</a>, and (3) the BM25 model and <a href=https://en.wikipedia.org/wiki/Unified_Modeling_Language>UMLS concepts</a> for retrieving the ideal answers (i.e., paragraph-sized summaries). Preliminary results show that our <a href=https://en.wikipedia.org/wiki/System>system</a> achieves good and competitive results in both exact and ideal answers extraction tasks as compared with the participating systems.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2338.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2338 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2338 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2338/>Adapting Pre-trained Word Embeddings For Use In Medical Coding</a></strong><br><a href=/people/k/kevin-patel/>Kevin Patel</a>
|
<a href=/people/d/divya-patel/>Divya Patel</a>
|
<a href=/people/m/mansi-golakiya/>Mansi Golakiya</a>
|
<a href=/people/p/pushpak-bhattacharyya/>Pushpak Bhattacharyya</a>
|
<a href=/people/n/nilesh-birari/>Nilesh Birari</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2338><div class="card-body p-3 small">Word embeddings are a crucial component in modern <a href=https://en.wikipedia.org/wiki/Natural_language_processing>NLP</a>. Pre-trained embeddings released by different groups have been a major reason for their popularity. However, they are trained on generic corpora, which limits their direct use for domain specific tasks. In this paper, we propose a method to add task specific information to pre-trained word embeddings. Such <a href=https://en.wikipedia.org/wiki/Information>information</a> can improve their utility. We add information from medical coding data, as well as the first level from the hierarchy of ICD-10 medical code set to different pre-trained word embeddings. We adapt CBOW algorithm from the <a href=https://en.wikipedia.org/wiki/Word2vec>word2vec package</a> for our purpose. We evaluated our approach on five different pre-trained word embeddings. Both the original word embeddings, and their modified versions (the ones with added information) were used for automated review of medical coding. The modified <a href=https://en.wikipedia.org/wiki/Word_embedding>word embeddings</a> give an improvement in <a href=https://en.wikipedia.org/wiki/F-score>f-score</a> by 1 % on the 5-fold evaluation on a private medical claims dataset. Our results show that adding extra information is possible and beneficial for the task at hand.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2340.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2340 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2340 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"><a class="badge badge-secondary align-middle mr-1 pwc-reduced-padding" href="https://paperswithcode.com/paper/?acl=W17-2340" data-toggle=tooltip data-placement=top title=Code><svg xmlns="http://www.w3.org/2000/svg" class="pwc-icon-small" viewBox="0 0 512 512"><path stroke="#4d8093" fill="#4d8093" d="M88 128h48v256H88zm144 0h48v256h-48zm-72 16h48v224h-48zm144 0h48v224h-48zm72-16h48v256h-48z"/><path stroke="#4d8093" fill="#4d8093" d="M104 104V56H16v4e2h88v-48H64V104zM408 56v48h40v304h-40v48h88V56z"/></svg></a></span><span class=d-block><strong><a class=align-middle href=/W17-2340/>Biomedical Event Trigger Identification Using Bidirectional Recurrent Neural Network Based Models</a></strong><br><a href=/people/r/rahul-v-s-s-patchigolla/>Rahul V S S Patchigolla</a>
|
<a href=/people/s/sunil-sahu/>Sunil Sahu</a>
|
<a href=/people/a/ashish-anand/>Ashish Anand</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2340><div class="card-body p-3 small">Biomedical events describe complex interactions between various biomedical entities. Event trigger is a word or a phrase which typically signifies the occurrence of an event. Event trigger identification is an important first step in all event extraction methods. However many of the current approaches either rely on complex hand-crafted features or consider features only within a window. In this paper we propose a method that takes the advantage of <a href=https://en.wikipedia.org/wiki/Recurrent_neural_network>recurrent neural network (RNN)</a> to extract higher level features present across the sentence. Thus hidden state representation of RNN along with word and entity type embedding as <a href=https://en.wikipedia.org/wiki/Feature_(machine_learning)>features</a> avoid relying on the complex hand-crafted features generated using various NLP toolkits. Our experiments have shown to achieve state-of-art F1-score on Multi Level Event Extraction (MLEE) corpus. We have also performed category-wise analysis of the result and discussed the importance of various <a href=https://en.wikipedia.org/wiki/Feature_(machine_learning)>features</a> in trigger identification task.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2341.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2341 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2341 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2341/>Representations of Time Expressions for Temporal Relation Extraction with <a href=https://en.wikipedia.org/wiki/Convolutional_neural_network>Convolutional Neural Networks</a></a></strong><br><a href=/people/c/chen-lin/>Chen Lin</a>
|
<a href=/people/t/timothy-miller/>Timothy Miller</a>
|
<a href=/people/d/dmitriy-dligach/>Dmitriy Dligach</a>
|
<a href=/people/s/steven-bethard/>Steven Bethard</a>
|
<a href=/people/g/guergana-savova/>Guergana Savova</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2341><div class="card-body p-3 small">Token sequences are often used as the input for Convolutional Neural Networks (CNNs) in <a href=https://en.wikipedia.org/wiki/Natural_language_processing>natural language processing</a>. However, they might not be an ideal representation for time expressions, which are long, highly varied, and semantically complex. We describe a <a href=https://en.wikipedia.org/wiki/Method_(computer_programming)>method</a> for representing time expressions with single pseudo-tokens for CNNs. With this <a href=https://en.wikipedia.org/wiki/Methodology>method</a>, we establish a new state-of-the-art result for a clinical temporal relation extraction task.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2342.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2342 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2342 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2342/>Automatic Diagnosis Coding of Radiology Reports : A Comparison of <a href=https://en.wikipedia.org/wiki/Deep_learning>Deep Learning</a> and Conventional Classification Methods</a></strong><br><a href=/people/s/sarvnaz-karimi/>Sarvnaz Karimi</a>
|
<a href=/people/x/xiang-dai/>Xiang Dai</a>
|
<a href=/people/h/hamed-hassanzadeh/>Hamed Hassanzadeh</a>
|
<a href=/people/a/anthony-nguyen/>Anthony Nguyen</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2342><div class="card-body p-3 small">Diagnosis autocoding services and research intend to both improve the productivity of clinical coders and the accuracy of the coding. It is an important step in <a href=https://en.wikipedia.org/wiki/Data_analysis>data analysis</a> for funding and reimbursement, as well as health services planning and <a href=https://en.wikipedia.org/wiki/Resource_allocation>resource allocation</a>. We investigate the applicability of <a href=https://en.wikipedia.org/wiki/Deep_learning>deep learning</a> at autocoding of radiology reports using International Classification of Diseases (ICD). Deep learning methods are known to require <a href=https://en.wikipedia.org/wiki/Big_data>large training data</a>. Our goal is to explore how to use these <a href=https://en.wikipedia.org/wiki/Methodology>methods</a> when the training data is sparse, skewed and relatively small, and how their effectiveness compares to conventional methods. We identify optimal parameters that could be used in setting up a <a href=https://en.wikipedia.org/wiki/Convolutional_neural_network>convolutional neural network</a> for <a href=https://en.wikipedia.org/wiki/Autocoding>autocoding</a> with comparable results to that of conventional methods.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2343.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2343 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2343 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2343/>Automatic classification of doctor-patient questions for a virtual patient record query task</a></strong><br><a href=/people/l/leonardo-campillos-llanos/>Leonardo Campillos Llanos</a>
|
<a href=/people/s/sophie-rosset/>Sophie Rosset</a>
|
<a href=/people/p/pierre-zweigenbaum/>Pierre Zweigenbaum</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2343><div class="card-body p-3 small">We present the work-in-progress of automating the classification of doctor-patient questions in the context of a simulated consultation with a virtual patient. We classify questions according to the computational strategy (rule-based or other) needed for looking up data in the <a href=https://en.wikipedia.org/wiki/Medical_record>clinical record</a>. We compare &#8216;traditional&#8217; machine learning methods (Gaussian and Multinomial Naive Bayes, and Support Vector Machines) and a neural network classifier (FastText). We obtained the best results with the SVM using <a href=https://en.wikipedia.org/wiki/Semantic_annotation>semantic annotations</a>, whereas the neural classifier achieved promising results without it.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2345.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2345 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2345 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2345/>Clinical Event Detection with Hybrid Neural Architecture</a></strong><br><a href=/people/a/adyasha-maharana/>Adyasha Maharana</a>
|
<a href=/people/m/meliha-yetisgen-yildiz/>Meliha Yetisgen</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2345><div class="card-body p-3 small">Event detection from clinical notes has been traditionally solved with rule based and statistical natural language processing (NLP) approaches that require extensive domain knowledge and feature engineering. In this paper, we have explored the feasibility of approaching this task with <a href=https://en.wikipedia.org/wiki/Recurrent_neural_network>recurrent neural networks</a>, clinical word embeddings and introduced a hybrid architecture to improve detection for entities with smaller representation in the dataset. A comparative analysis is also done which reveals the complementary behavior of <a href=https://en.wikipedia.org/wiki/Neural_network>neural networks</a> and <a href=https://en.wikipedia.org/wiki/Conditional_random_field>conditional random fields</a> in clinical entity detection.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2346.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2346 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2346 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2346/>Extracting Personal Medical Events for User Timeline Construction using Minimal Supervision</a></strong><br><a href=/people/a/aakanksha-naik/>Aakanksha Naik</a>
|
<a href=/people/c/christopher-bogart/>Chris Bogart</a>
|
<a href=/people/c/carolyn-rose/>Carolyn Rose</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2346><div class="card-body p-3 small">In this paper, we describe a <a href=https://en.wikipedia.org/wiki/System>system</a> for automatic construction of user disease progression timelines from their posts in online support groups using minimal supervision. In recent years, several online support groups have been established which has led to a huge increase in the amount of patient-authored text available. Creating systems which can automatically extract important medical events and create disease progression timelines for users from such text can help in patient health monitoring as well as studying links between <a href=https://en.wikipedia.org/wiki/Disease>medical events</a> and users&#8217; participation in <a href=https://en.wikipedia.org/wiki/Support_group>support groups</a>. Prior work in this domain has used manually constructed keyword sets to detect medical events. In this work, our aim is to perform medical event detection using minimal supervision in order to develop a more general timeline construction system. Our <a href=https://en.wikipedia.org/wiki/System>system</a> achieves an <a href=https://en.wikipedia.org/wiki/Accuracy_and_precision>accuracy</a> of 55.17 %, which is 92 % of the performance achieved by a supervised baseline system.</div></div><p class="d-sm-flex align-items-stretch"><span class="d-block mr-2 text-nowrap list-button-row"><a class="badge badge-primary align-middle mr-1" href=https://aclanthology.org/W17-2348.pdf data-toggle=tooltip data-placement=top title="Open PDF">pdf
</a><a class="badge badge-info align-middle mr-1" href=#abstract-W17-2348 data-toggle=collapse aria-expanded=false aria-controls=abstract-W17-2348 title="Show Abstract">abs</a><br class="d-none d-sm-inline-block"></span><span class=d-block><strong><a class=align-middle href=/W17-2348/>A Multi-strategy Query Processing Approach for Biomedical Question Answering : USTB_PRIR at BioASQ 2017 Task 5B<span class=acl-fixed-case>USTB</span>_<span class=acl-fixed-case>PRIR</span> at <span class=acl-fixed-case>B</span>io<span class=acl-fixed-case>ASQ</span> 2017 Task 5<span class=acl-fixed-case>B</span></a></strong><br><a href=/people/z/zan-xia-jin/>Zan-Xia Jin</a>
|
<a href=/people/b/bo-wen-zhang/>Bo-Wen Zhang</a>
|
<a href=/people/f/fan-fang/>Fan Fang</a>
|
<a href=/people/l/le-le-zhang/>Le-Le Zhang</a>
|
<a href=/people/x/xu-cheng-yin/>Xu-Cheng Yin</a></span></p><div class="card bg-light mb-2 mb-lg-3 collapse abstract-collapse" id=abstract-W17-2348><div class="card-body p-3 small">This paper describes the participation of USTB_PRIR team in the 2017 BioASQ 5B on <a href=https://en.wikipedia.org/wiki/Question_answering>question answering</a>, including <a href=https://en.wikipedia.org/wiki/Document_retrieval>document retrieval</a>, snippet retrieval, and concept retrieval task. We introduce different multimodal query processing strategies to enrich query terms and assign different weights to them. Specifically, sequential dependence model (SDM), pseudo-relevance feedback (PRF), fielded sequential dependence model (FSDM) and Divergence from Randomness model (DFRM) are respectively performed on different fields of PubMed articles, sentences extracted from relevant articles, the five terminologies or ontologies (MeSH, GO, Jochem, Uniprot and DO) to achieve better search performances. Preliminary results show that our <a href=https://en.wikipedia.org/wiki/System>systems</a> outperform others in the document and snippet retrieval task in the first two batches.</div></div></div></section></div><footer class="bg-gradient-dark py-2 py-xl-3 mt-3 mt-md-4 mt-xl-5" style=color:#fff><div class=container><p class="small px-1"><span class="float-right mt-2 ml-2"><a rel=license href=http://creativecommons.org/licenses/by/4.0/><img alt="Creative Commons License" style=border-width:0 src=https://i.creativecommons.org/l/by/4.0/88x31.png></a></span>
ACL materials are Copyright ©&nbsp;1963&ndash;2022 ACL; other materials are copyrighted by their respective copyright holders. Materials prior to 2016 here are licensed under the <a href=https://creativecommons.org/licenses/by-nc-sa/3.0/>Creative Commons Attribution-NonCommercial-ShareAlike 3.0 International License</a>. Permission is granted to make copies for the purposes of teaching and research. Materials published in or after 2016 are licensed on a <a href=https://creativecommons.org/licenses/by/4.0/>Creative Commons Attribution 4.0 International License</a>.</p><p class="small px-1">The ACL Anthology is managed and built by the <a href=/info/credits/>ACL Anthology team</a> of volunteers.</p><p class="small px-1"><i>Site last built on 23 May 2022 at 01:26 UTC with <a href=https://github.com/acl-org/acl-anthology/tree/6b5537cdc480294721047e4e67eca5cb2bec4ab4>commit 6b5537cd</a>.</i></p></div></footer><script src=https://code.jquery.com/jquery-3.3.1.slim.min.js integrity=sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo crossorigin=anonymous></script>
<script src=https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js integrity=sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut crossorigin=anonymous></script>
<script src=https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js integrity=sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k crossorigin=anonymous></script>
<script>$(function(){$('[data-toggle="tooltip"]').tooltip(),$("#toggle-all-abstracts")&&($("#toggle-all-abstracts").click(function(){var e=$("#toggle-all-abstracts");e.attr("disabled",!0),e.attr("data-toggle-state")=="hide"?($(".abstract-collapse").collapse("show"),e.attr("data-toggle-state","show")):($(".abstract-collapse").collapse("hide"),e.attr("data-toggle-state","hide")),e.attr("disabled",!1)}),$("#toggle-all-abstracts").attr("disabled",!1))})</script></body></html>